src/pkg/utils/utils.go

```diff
@@ -500,9 +500,12 @@ func HumanDuration(duration int64) string {
 }
 
 // ImageReferenceCanBeID checks if 'image' might be the ID of an image
-func ImageReferenceCanBeID(image string) (bool, error) {
-	matched, err := regexp.MatchString("^[a-f0-9]\\{6,64\\}$", image)
-	return matched, err
+func ImageReferenceCanBeID(image string) bool {
+	matched, err := regexp.MatchString("^[a-f0-9]{6,64}$", image)
+	if err != nil {
+		logrus.Debugf("failed to match regexp for ID reference: %v", err)
```

Comment:
I'd use a `panic` here. An error can only be thrown if the regular expression is faulty, which would be a programmer error, not a run-time condition.

Reply:
Fixed.